### ISSUE TITLE
2.11.1 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3651]
- #3594 
- #3579
- #3578
- #3605 
- #3606
- #3604 
- #3609 
- #3607 
- #3608 
- #3573 
- #3574 
- #3601
- #3615 
- #3613 
- #3611 
- #3620
- #3624
- #3617
- #3625
- #3593 
- #3626 
- #3633 
- #3621
- #3636 
- #3629 
- #3630 
- #3628
- #3632 
- #3640 
- #3614 
- #3641 
- #3643
- #3635
- #3648 

```
git cherry-pick 731a8fe907674b97a9407ee59f6c488626b3881b
git cherry-pick e3d10bec742335efbad5b8b09e4d5a82181ceba7
git cherry-pick f28f65878f413e11b5fdb76341aae42e58f425e1
git cherry-pick 6fcbd7e5cdac322d543e44f18ccdf2970d7f49d6
git cherry-pick 6218c50cc2fa30bb118431bcc2ff5f26c1c87266
git cherry-pick a00510ddb30c1a341911628757de696fa6746ce4
git cherry-pick 99c00d49be696571e022d0eabc77da6ef1f798ff
git cherry-pick 1673f037a4e8d5af1cd861dc025f43147a8d3250
git cherry-pick ac37082829118837018293eca5164a214a55672f
git cherry-pick ed8b243376065bf8f1a320cbdc38f5eb75879b40
git cherry-pick 669158456d56bfc10bf2f3ada34c0910efc4e4b4
git cherry-pick 1956c31a4010103d5d5617c38776f47a67e2f063
git cherry-pick b16dda5b97b15681e639f23ffa68a728c7c4dc62
git cherry-pick 0af015acac2a2421fff424828ad95631741b739e
git cherry-pick 7706acd7dd51cb128dd5cec3249b4b37f0b4f146
git cherry-pick fd811d0190a943bfd72b23fa41cdf09464300c16
git cherry-pick 2e064fe71694b8bb90eee1b11bd45d25ed3a2d92
git cherry-pick db3abf4b67f74397f56cc44721b68eeda597d3ff
git cherry-pick 2011fde9a587eb6a044a3248a6b1d05dfdb3d00a
git cherry-pick 26a8f96a149839748e20e73a6113cca65453e417
git cherry-pick 5a4db4dcb31b13d58888ce482d95ecc579a5de2d
git cherry-pick db92fd8df08fd7463f9ef73d86932df907a18275
git cherry-pick 8884a7dff08546d49700ec436d389fed9ec1b71e
git cherry-pick efce0c5aa7143b4a4539f526f7fd6ee19a72a5c1
git cherry-pick f75cb28005c6981d1fd4b4c639655cfefc9c1730
git cherry-pick 928ed380dec67edee33da91668674fc86254d96e
git cherry-pick 82bc5bc970c808c9d10e17e586ca4fd9234a56e8
git cherry-pick 91c8e0c2c5f115612290ca39237740a0fb2e342c
git cherry-pick cdbdc3d5e47877530c4db8f585541a93e4221cc6
git cherry-pick 21cefc3bc3ff3f2a5b1a8a3b895d24372c6dba40
git cherry-pick e7fd8bab58578eca4c806add538129533af44295
git cherry-pick 8ac880e3a2b320fd1059a153b312f71c95e089cb
git cherry-pick 5a681aba29e0d564bd504e9b0e8d3a52f05c1537
git cherry-pick 3184dd21ba45adf60bbc11ba5ded6d1b95a75842
```

**Comment by bberndt-uaz** [comment]
I believe this one should be included as well? https://github.com/az-digital/az_quickstart/pull/3636

**Comment by bberndt-uaz** [comment]
This was added to the 2.12.0-alpha1 project, but the PR updating the module to 2.0.2 is included with 2.11.1 so I'll change this one to be in the 2.11.1 project as well:

- #3617

Not added to any project, I'll add it to the 2.11.1 project:

- #3626

This was added to the 2.12.0-alpha1 project and I'll leave it up to you if we want to move it to the 2.11.1 project:

- #3406

**Comment by joeparsons** [comment]
> This was added to the 2.12.0-alpha1 project, but the PR updating the module to 2.0.2 is included with 2.11.1 so I'll change this one to be in the 2.11.1 project as well:
> 
> * [Update drupal/extlink requirement from 1.7 to 2.0.1 #3617](https://github.com/az-digital/az_quickstart/pull/3617)
> 
> Not added to any project, I'll add it to the 2.11.1 project:
> 
> * [Update drupal/blazy requirement from 3.0.9 to 3.0.10 #3626](https://github.com/az-digital/az_quickstart/pull/3626)
> 
> This was added to the 2.12.0-alpha1 project and I'll leave it up to you if we want to move it to the 2.11.1 project:
> 
> * [Update drupal/workbench_access requirement from 2.0.1 to 2.0.3 #3406](https://github.com/az-digital/az_quickstart/pull/3406)

Thanks @bberndt-uaz!

I think I'll remove the workbench_access one from this release.  My bad.

**Comment by joeparsons** [comment]
> I believe this one should be included in the plan as well?
> 
> * [Update drupal/block_field requirement from 1.0.0-rc4 to 1.0.0-rc5 #3636](https://github.com/az-digital/az_quickstart/pull/3636)

Thanks.  I'll add this.

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.11.0..HEAD --pretty='format:- %s'`
```
- #3647: Disable az_seo_update_1021001 and az_paragraphs_views_side_by_side_update_1021001. (#3648)
- Update drupal/easy_breadcrumb requirement from 2.0.7 to 2.0.8 (#3635)
- Closes #3642 Upgrade Drupal core to 10.3.2 (#3643)
- Update drupal/smart_date requirement from 4.1.4 to 4.1.5 (#3641)
- Update drupal/paragraphs requirement from 1.17.0 to 1.18.0 (#3614)
- Fixes #3639 Order Authors by Name (#3640)
- Update drupal/auto_entitylabel requirement from 3.2.0 to 3.3.0 (#3632)
- Update drupal/honeypot requirement from 2.1.3 to 2.1.4 (#3628)
- Update drupal/redirect requirement from 1.9.0 to 1.10.0 (#3630)
- Update drupal/slick requirement from 3.0.2 to 3.0.3 (#3629)
- Update drupal/block_field requirement from 1.0.0-rc4 to 1.0.0-rc5 (#3636)
- Update drupal/intelligencebank requirement from 4.0.0 to 4.1.0 (#3621)
- Update drupal/extlink requirement from 2.0.1 to 2.0.2 (#3633)
- Update drupal/blazy requirement from 3.0.9 to 3.0.10 (#3626)
- Update drupal/pathauto requirement from 1.12.0 to 1.13.0 (#3593)
- Update citation-style-language/styles requirement from 0.0.442 to 0.0.447 (#3625)
- Update drupal/extlink requirement from 1.7 to 2.0.1 (#3617)
- Update drupal/embed requirement from 1.7.0 to 1.9.0 (#3624)
- Update drupal/masquerade requirement from 2.0.0-rc5 to 2.0.0 (#3620)
- Update drupal/menu_block requirement from 1.11.0 to 1.13.0 (#3611)
- Update drupal/views_bootstrap requirement from 5.4.0 to 5.4.1 (#3613)
- Update drupal/asset_injector requirement from 2.20.0 to 2.21.0 (#3615)
- Update citation-style-language/styles requirement from 0.0.434 to 0.0.442 (#3601)
- Update drupal/google_tag requirement from 2.0.5 to 2.0.6 (#3574)
- Update drupal/inline_entity_form requirement from 3.0.0-rc19 to 3.0.0-rc20 (#3573)
- Update drupal/masquerade requirement from 2.0.0-rc4 to 2.0.0-rc5 (#3608)
- Update drupal/metatag requirement from 2.0.0 to 2.0.2 (#3607)
- Update drupal/viewsreference requirement from 2.0.0-beta8 to 2.0.0-beta9 (#3609)
- Update drupal/config_ignore_readonly requirement from 2.2.0-beta1 to 2.2.0 (#3604)
- Update drupal/date_ap_style requirement from 2.0.0 to 2.0.1 (#3606)
- Update drupal/field_group requirement from 3.4.0 to 3.6.0 (#3605)
- Update drupal/smtp requirement from 1.3.0 to 1.4.0 (#3578)
- Update drupal/token requirement from 1.14.0 to 1.15.0 (#3579)
- Update drupal/smart_date requirement from 4.1.3 to 4.1.4 (#3594)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.11.1
